app/models/vote.rb

```diff
@@ -0,0 +1,5 @@
+    class Vote < ActiveRecord::Base
+      belongs_to :user
+      belongs_to :votable, :polymorphic =>true
+      validates :votable_type, :votable_id, :presence => true
```

Comment:
I assume this is from a gem? You'll want to update this to reflect current Ruby syntax: presence: true, polymorphic: true